en/features/verification.md

```diff
@@ -0,0 +1,195 @@
+
+<tr>
+<td><a href="/{{page.lang}}/features/details/chain-rewrites/">Chain rewrites</a></td>
+  <td class="bgred"></td>
+  <td class="bgred"></td>
+  <td class="bgred"></td>
+</tr>
+
+</table>
+
+Note that although all programs---including Bitcoin Core---are
+vulnerable to chain rewrites, Bitcoin provides a defense mechanism: the
+more confirmations your transactions have, the safer you are. *There is
+no known decentralized defense better than that.*
+
+## Help Protect Decentralization
+
+{% comment %}<!-- 1231006505 is the time in block 0; 31558149 is the average number of seconds in a year -->{% endcomment %}
+{% capture bitcoin_age %}{{ site.time | date: "%s" | minus: "1231006505" | divided_by: "31558149" }}{% endcapture %}
+
+The bitcoin currency only works when people accept bitcoins in exchange
+for other valuable things. That means it's the people accepting
+bitcoins who give it value and who get to decide how Bitcoin should work.
+
+When you accept bitcoins, you have the power to enforce Bitcoin's rules,
+such as preventing confiscation of any person's bitcoins without access
+to that person's private keys.
+
+Unfortunately, many users outsource their enforcement power. This
+leaves Bitcoin's decentralization in a weakened state where a handful of
+miners can collude with a handful of service providers to change
+Bitcoin's rules for all those non-verifying users who outsourced their power.
+
+<table class="received_transactions center">
+  <tr>
+    <td class="center" markdown="span">*Users of wallet services*<br
+      >**Trust service providers**{:.fgred}</td>
+
+    <td class="center" markdown="span">*Users of P2P light wallets*<br
+    >**Trust miners**{:.fgred}</td>
+  </tr>
+
+  <tr>
+    <td class="center" markdown="span">*Users of client light wallets*<br
+    > **Trust "free" servers**{:.fgred}</td>
+
+    <td class="center" markdown="span">*Users of Bitcoin Core*<br
+    >**Enforce the rules**{:.fggreen}</td>
+  </tr>
+</table>
+
+Unlike other wallets, **Bitcoin Core *does* enforce the rules**---so
+if the miners and service providers change the rules for their non-verifying
+users, those users will be unable to pay full verification Bitcoin Core
+users like you.
+
+As long as there are many non-verifying users who want to be able to
+pay Bitcoin Core users, miners and others know they can't effectively
+change Bitcoin's rules.
+
+But what if not enough non-verifying users care about paying Bitcoin
+Core users? Then it becomes easy for miners and service providers to
+take control of Bitcoin, likely bringing to an end this {{bitcoin_age}}
+year experiment in decentralized currency.
+
+{:.center}
+![History of Bitcoin](/assets/images/losing-decentralization.png)
+
+If you think **Bitcoin should remain decentralized,** the best thing you
+can do is [verify every payment you receive](#do-you-verify) using your own personal
+full node such as Bitcoin Core.
+
+We don't know how many full verification users and business are needed,
```

Comment:
```suggestion
We don't know how many fully-verifying users and business are needed,
```

Reply:
What's the motivation for these changes from "full verification" to "fully verifying"?  I think finding as many ways to repeat "full verification" without getting excessive is probably a good way to firmly get this concept in the readers brain.  Rephrasing the concept waters it down IMO.

Reply:
You can't have a "verification user".  You can have a "verifying user".  That is the adjective form.

Reply:
Does that mean you also can't have a "bitcoin user" but only a "bitcoining user"?

Sorry for being snarky, but I'm not aware of the rule you're referring to.  I'd prefer to keep this text as-is.

Reply:
It's a good question.

Bitcoin is a proper noun, like "Internet".  You are an" Internet user".  But then again, we are not a grammar text book, so if you strongly prefer the other way, that's fine.